scripts/run_agent_open_sinoptik_real.py

```
@@ -0,0 +1,105 @@
+            return {"reflection": "open sinoptik"}, [code, "done"]
+        else:
+            return {"reflection": "done"}, ["done"]
+
+
+def fetch_month_block(url):
+    tried = []
+    for u in URL_CANDIDATES:
+        try:
+            # Ensure proper quoting for non-ascii parts
+            u_quoted = u if all(ord(c) < 128 for c in u) else quote(u, safe=':/')
+            resp = urllib.request.urlopen(u_quoted, timeout=10)
```

Ruff (0.14.10)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)